src/Native/MklProxyNative/MklProxyNative.cpp

```diff
@@ -0,0 +1,176 @@
+       for C] */
+    OutputStrides = 13,
+
+    /* Distance between first input elements for multiple transforms
+       [0] */
+    InputDistance = 14,
+
+    /* Distance between first output elements for multiple transforms
+       [0] */
+    OutputDistance = 15,
+
+    /* Effort spent in initialization [Medium] */
+    /* InitializationEffort = 16, ## NOT IMPLEMENTED */
+
+    /* Use of workspace during computation [Allow] */
+    /* Workspace = 17, ## NOT IMPLEMENTED */
+
+    /* Ordering of the result [Ordered] */
+    Ordering = 18,
+
+    /* Possible transposition of result [None] */
+    Transpose = 19,
+
+    /* User-settable descriptor name [""] */
+    DescriptorName = 20, /* DEPRECATED */
+
+    /* Packing format for ComplexReal storage of finite
+       conjugate-even sequences [CcsFormat] */
+    PackedFormat = 21,
+
+    /* Commit status of the descriptor - R/O parameter */
+    CommitStatus = 22,
+
+    /* Version string for this DFTI implementation - R/O parameter */
+    Version = 23,
+
+    /* Ordering of the forward transform - R/O parameter */
+    /* ForwardOrdering  = 24, ## NOT IMPLEMENTED */
+
+    /* Ordering of the backward transform - R/O parameter */
+    /* BackwardOrdering = 25, ## NOT IMPLEMENTED */
+
+    /* Number of user threads that share the descriptor [1] */
+    NumberOfUserThreads = 26
+};
+
+enum ConfigValue
+{
+    /* CommitStatus */
+    Committed = 30,
+    Uncommitted = 31,
+
+    /* ForwardDomain */
+    Complex = 32,
+    Real = 33,
+    /* ConjugateEven = 34,   ## NOT IMPLEMENTED */
+
+    /* Precision */
+    Single = 35,
+    Double = 36,
+
+    /* ForwardSign */
+    /* Negative = 37,         ## NOT IMPLEMENTED */
+    /* Positive = 38,         ## NOT IMPLEMENTED */
+
+    /* ComplexStorage and ConjugateEvenStorage */
+    ComplexComplex = 39,
+    ComplexReal = 40,
+
+    /* RealStorage */
+    RealComplex = 41,
+    RealReal = 42,
+
+    /* Placement */
+    InPlace = 43,          /* Result overwrites input */
+    NotInPlace = 44,      /* Have another place for result */
+
+    /* InitializationEffort */
+    /* Low = 45,              ## NOT IMPLEMENTED */
+    /* Medium = 46,           ## NOT IMPLEMENTED */
+    /* High = 47,             ## NOT IMPLEMENTED */
+
+    /* Ordering */
+    Ordered = 48,
+    BackwardScrambled = 49,
+    /* ForwardScrambled = 50, ## NOT IMPLEMENTED */
+
+    /* Allow/avoid certain usages */
+    Allow = 51,            /* Allow transposition or workspace */
+    /* Avoid = 52,            ## NOT IMPLEMENTED */
+    None = 53,
+
+    /* PackedFormat (for storing congugate-even finite sequence
+       in real array) */
+    CcsFormat = 54,       /* Complex conjugate-symmetric */
+    PackFormat = 55,      /* Pack format for real DFT */
+    PermFormat = 56,      /* Perm format for real DFT */
+    CceFormat = 57        /* Complex conjugate-even */
+};
+
+EXPORT_API(int) DftiSetValue(void *handle, ConfigParam config_param, ...);
```

Comment:
It would be nice having a comment here explaining why we need to make these "proxy" methods.